index.html

```diff
@@ -0,0 +1,133 @@
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Global Summit</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link href="https://fonts.cdnfonts.com/css/lato" rel="stylesheet">
+</head>
+
+<body>
+    <header>
+        <div class="mobile-navbar">
+            <nav class="mobile-navbar-container">
+                <span class="hamburger-icon"><img src="images/hamburger.png" alt="hamburger icon"></span>
+                <img src="images/Icon.png" class="close-icon-container" alt="close icon">
+                <ul class="nav-links">
+                    <li><a href="/about.html">About</a></li>
+                    <li><a href="/about.html">Schedule</a></li>
+                    <li><a href="/about.html">Join</a></li>
+                    <li><a href="/about.html">Sponsor</a></li>
+                    <li><a href="/about.html">News</a></li>
+                    <li><a href="/about.html">CC Campaign</a></li>
+                </ul>
+            </nav>
+            <a href="#"><img src="images/header-logo.png" alt="Global Summit Logo"></a>
+        </div>
+
+        <ul class="navbar-banner">
+            <li><a href="#"><img src="images/facebook-logo.png" class="mini-navbar-img" alt="facebook logo"></a></li>
+            <li><a href="https://twitter.com/AyokunnumiA"><img src="images/twitter-logo.png" class="mini-navbar-img"
+                        alt="twitter logo"></a></li>
+            <li><a href="#">English</a></li>
+            <li><a href="#">My Logo</a></li>
+            <li><a href="#">Logout</a></li>
+        </ul>
+
+        <div>
+            <nav class="desktop-navbar">
+                <div><img src="images/header-logo.png" alt=""></div>
+                <ul class="desktop-navbar-list">
+                    <li><a href="/about.html" class="desktop-navbar-link">About</a></li>
+                    <li><a href="#" class="desktop-navbar-link">Program</a></li>
+                    <li><a href="#" class="desktop-navbar-link">Join</a></li>
+                    <li><a href="#" class="desktop-navbar-link">Sponsor</a></li>
+                    <li><a href="#" class="desktop-navbar-link">News</a></li>
+                    <li><a href="#" class="desktop-navbar-link CC-Campaign-link">CC Campaign</a></li>
+                </ul>
+            </nav>
+        </div>
+
+        <div class="desktop-header-section">
+            <div class="mobile-header-section">
+                <p class="header-theme">"You Never Fail Until You Stop Trying"</p>
+                <h1>Content Creator <br> Global Summit 2023</h1>
+                <p class="header-hero-desc">A joyful celebration believing in the value of openness and
+                    sharing, creating a positive change with people from over 80
+                    countries is taking place in October, in Korea.
+                </p>
```

Comment:
- [ ] As shown in the project design below the text are much larger than what you currently have on your design, i will kindly suggest that your increase the text size to match closely the provided project design 👍 
- project design
![ht](https://github.com/Ayokunnumi1/Conference-Page/assets/89055502/5ad3fe33-40b1-4193-8c05-e9b198d5019e)
- your design
![hts](https://github.com/Ayokunnumi1/Conference-Page/assets/89055502/dc279d7e-6e13-46c4-a6b9-6e91940cb55f)